### DISCUSSION TITLE
Multiple webcams

### DISCUSSION THREAD

**Discussion by Makunugi** [cables-gl/cables_docs#251] [Answers & Questions] [answered]
Hello.
Is it possible to display two or more webcams at the same time?

**Comment by positlabs** [comment, marked as answer]
Totally. You would just need to use separate instances of the webcam op, and change the device ID. Here's a patch that uses a webcam op from dev that enables this kind of thing. https://cables.gl/edit/6kk1nZ

**Reply by Makunugi** [reply]
very nice!
I'm still a beginner and couldn't imagine a patch so far.
I am going to use it as an example.
thank you very much!
